peekingduck/configs/draw/bbox.yml

```diff
@@ -1,5 +1,4 @@
 input: ["bboxes", "img"]
```

Comment:
It would be more consistent to include this?

```suggestion
input: ["bboxes", "img", "bbox_labels"]
```

Reply:
See below discussion regarding this issue

Reply:
Following below discuss, add bbox labels